package.json

```
@@ -29,7 +29,8 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:watch": "jest --watch",
-    "prettier:fix": "prettier --write --ignore-path .gitignore ."
+    "prettier:fix": "prettier --write --ignore-path .gitignore .",
+    "test-storybook": "test-storybook"
```

To follow the pattern used in other scripts.

```suggestion
    "test:storybook": "test-storybook"
```

---

Alterado